src/antidote_crdt_orset.erl

```diff
@@ -108,33 +108,33 @@ value({fragment, Elem}, ORSet) ->
 value({tokens, Elem}, ORSet) ->
     case orddict:find(Elem, ORSet) of
         error ->
-            orddict:new();
+            [];
```

Comment:
What does `Token::any()` mean?
I just checked http://erlang.org/doc/man/crypto.html#strong_rand_bytes-1
Does it make sense to use `binary()` here?